### ISSUE TITLE
Content update for "Getting to work" section: Deadline extended

### ISSUE THREAD

**Issue by esquared415** [cal-itp/website#40]
Hello! As soon as you're able, we need a sentence replaced to reflect a deadline change.
- [ ] In the 4th paragraph under "Getting to work" section, please change _last line_ to:
Interested parties are asked to click the above link, read the instructions, and respond by no later than 3:00 p.m. (PDT) on April 15th, 2022 (we've extended the deadline from April 1st).

Thanks @thekaveman and team!

**Comment by thekaveman** [comment]
Working on this now @esquared415!

Just to be sure, do you want to include the last bit in parenthesis ("we've extended...") on the page, or was that a note for me?

> Interested parties are asked to click the above link, read the instructions, and respond by no later than 3:00 p.m. (PDT) on April 15th, 2022 **(we've extended the deadline from April 1st)**.

**Comment by esquared415** [comment]
I had intended for that last parenthetical bit to be on the page (not just a note for you), as the PDF still reflects the old 4/1 deadline (and Regina did not want to change it there). But I approved it without that parenthetical. 

If there’s any confusion, we can stick the parenthetical in there. But we should be fine without it. 

Oddly, I didn’t see this correspondence/ query in the slack integration or the ticket (as I was looking at the review request). I just noticed it when I checked my GitHub-associated email. 

EE
